template.go

```diff
@@ -341,8 +341,40 @@ func writeDebianGbpConf(dir string, dep14, pristineTar bool) error {
 		fmt.Fprintf(f, "dist = DEP14\n")
 	}
 	if pristineTar {
-		fmt.Fprintf(f, "pristine-tar = True\n")
+		fmt.Fprintf(f, `
+# Always use pristine tar to improve supply chain security and auditability
```

Comment:
I don't agree with this statement, and that it improves security. pristine-tar is a hack, and the team policy to reject its use is the correct one, we should be removing all of its usage across the team not encouraging it again. This wording tries to give it a positive side which it does not have.

If a comment needs to be added, then it should spell the team policy and that pristine-tar is problematic and should be avoided.

Reply:
Pristine-tar is not a hack. If you claim that it is bad, you should present some evidence, and then we can assess if it is compelling. In https://salsa.debian.org/go-team/go-team.pages.debian.net/-/merge_requests/7#note_574266 I already asked for this, and you just provided a link to a comment in a bug report that is 10+ years old.

Reply:
Please stop using ad hominem arguments here and in your other comments elsewhere. The merits of using pristine-tar can fully be debated on its own without having to resort to ad hominem arguments about who is suggesting it.

I have been using pristine-tar on numerous packages for many years and it works well. The [man page is not stating](https://manpages.debian.org/unstable/pristine-tar/pristine-tar.1.en.html) it is a hack or discouraging it's use. It does it's job, allows one to run `gbp clone` of a repo when reviewing changes and building, and it is very space efficient. In the Debian packaging world where we require the package sources to include the upstream source, pristine-tar serves its purpose well.